packages/esm-commons-lib/src/components/encounter-tile/encounter-tile.test.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { act, render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
```

Comment:
we don't need this

Reply:
if i remove it i get undefined for some jest functions